features/printed_consent_forms.feature

```diff
@@ -13,6 +13,7 @@ Feature:
     And I should see links back to edit things that I provided
     When I click continue
     Then I should see the age-appropriate consent form preview
+    And it should have a place for name, signature and date
```

Comment:
Hmm, if the IDs aren't being used for anything else but testing, perhaps they should come out.